
<!DOCTYPE html>
<html>
<head>
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
<meta name="viewport" content="width=device-width, initial-scale=1">
<style>
.accordion {
  background-color: #a0dab3;
  color: #58595b;
  cursor: pointer;
  padding: 18px;
  width: 100%;
  border: none;
  text-align: left;
  outline: none;
  font-size: 15px;
  font-family: "Montserrat", sans-serif;
  font-weight: bold;
  border: 1px solid #bcbec0;
  transition: 0.4s;
}


.active, .accordion:hover {
  background-color: #00965e;
  color: white;
}

h2 {
  font-family: "Montserrat", sans-serif; 
}

p {
  font-family: "Montserrat", sans-serif;
}

.panel {
  padding: 0 18px;
  display: none;
  background-color: white;
  overflow: hidden;
}
</style>
</head>
<body>


<button class="accordion active">2026</button>
<div class="panel"  style="display: block;">
<p></p>
<button class="accordion active">2026-02-23 <br> Dr. Tianzhong Yang, University of Minnesota <br> Causal Models for High-Dimensional Omics Data</button>
<div class="panel" style="display: block;">
<p>This talk presents two statistical frameworks for elucidating disease mechanisms using high-dimensional omics data. In the first part, I will introduce pseudotime-dependent transcriptome-wide association studies (pt-TWAS), which integrate GWAS data with single-cell gene expression to identify genes and cell stages underlying disease risk. Pt-TWAS models gene expression as a continuous function of pseudotime and leverages shared genetic effects across cell stages. We show that pt-TWAS captures dynamic genetic effects along developmental trajectories and achieves higher statistical power than pseudobulk TWAS analysis where single-cell gene expression is averaged within each cell types. Applied to B-cell acute lymphoblastic leukemia, pt-TWAS replicates known risk genes and pinpoints their relevant developmental stages. In the second part, I will present a new total mediation effect measure for high-dimensional omics mediators in case-control studies. Existing high-dimensional mediation methods largely focus on continuous outcomes, with limited applicability to binary outcomes and case–control designs. Moreover, many rely on sparsity assumptions that are ill-suited for settings with numerous weak mediators and small, non-sparse effects. We propose a variance-based total mediation effect measure within a liability framework that avoids cancellation of opposing effects and is invariant to disease prevalence. To enable valid inference in case–control studies, we develop a cross-fitted, modified Haseman–Elston regression–based estimator that yields consistent estimation in the presence of non-mediators and weak effects, without requiring exact mediator selection. In an application to the Women’s Health Initiative, we find widespread weak mediation of the BMI–coronary heart disease association through metabolomics, with an estimated 89% of the explained liability mediated by measured metabolites.</p>
</div>
</div>

<script>
var acc = document.getElementsByClassName("accordion");
var i;

for (i = 0; i < acc.length; i++) {
  acc[i].addEventListener("click", function() {
    this.classList.toggle("active");
    var panel = this.nextElementSibling;
    if (panel.style.display === "block") {
      panel.style.display = "none";
    } else {
      panel.style.display = "block";
    }
  });
}
</script>

</body>
</html>

